lib/dl_task_processor/dl_task_processor/task.py

```diff
@@ -119,7 +117,7 @@ class Retry(TaskResult):
 
 @attr.s
 class BaseExecutorTask(Generic[_BASE_TASK_META_TV, _BASE_TASK_CONTEXT_TV], metaclass=abc.ABCMeta):
-    cls_meta: ClassVar[Type[_BASE_TASK_META_TV]]
+    cls_meta: ClassVar[Type[_BASE_TASK_META_TV]]  # type: ignore[misc]  # TODO: fix
```

Comment:
are we ignoring this one specifically to make dl_task_processor green on mypy? if not, we probably should fix it with other `ClassVat[Type[...]]` errors